### PR TITLE
qdirstat: 1.6.1 -> 1.7

### PR DESCRIPTION
**Pull request by Atemu**
###### Motivation for this change

https://github.com/shundhammer/qdirstat/releases/tag/1.7

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Works for me!

File: pkgs/applications/misc/qdirstat/default.nix [hunk 1]
```diff
@@ -3,7 +3,7 @@
 , makeWrapper, perlPackages, mkDerivation }:
 
 let
-  version = "1.6.1";
+  version = "1.7";
 in mkDerivation rec {
```
Comment (sikmir) [inline pkgs/applications/misc/qdirstat/default.nix:7]:
```suggestion
in mkDerivation {
```

Comment (gnidorah) [reply pkgs/applications/misc/qdirstat/default.nix:7]: I would do this instead
```nix
{ stdenv, fetchFromGitHub, qmake
, coreutils, xdg_utils, bash
, makeWrapper, perlPackages, mkDerivation }:

mkDerivation rec {
  pname = "qdirstat";
  version = "1.7";
```

Comment (sikmir) [reply pkgs/applications/misc/qdirstat/default.nix:7]: Generally `let ... in` is a bit better way over `rec`. See https://nix.dev/anti-patterns/language.html#rec-expression. But `mkDerivation rec {` is more widely used throw nixpkgs.

Comment (zowoq) [reply pkgs/applications/misc/qdirstat/default.nix]: > https://github.com/NixOS/nixpkgs/pull/93296#issuecomment-663763561
>
> https://nix.dev/anti-patterns/index.html is an "opinionated guide" that lists several patterns which are still allowed in this repo and some are used in almost all derivations.
>
> See https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293/2 for some discussion on this.

**Review by davidak — APPROVED** [review]
looks good
works

Screenshot from 2020-09-01 12-18-20

**Comment by timokau** [comment]
Thanks @Atemu for the update and @davidak for the review :)

**Pull request merged**